assets/account.css

```diff
@@ -227,16 +242,121 @@
   background: var(--background-accent);
 }
 
-.account__error-message {
-  font-size: 14px;
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)) {
+  color: var(--color-red);
+}
+
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)):before {
+  border-color: var(--color-red);
+  background: var(--color-red-light);
+}
+
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)):after {
+  border-color: var(--color-red);
+}
+
+.account-checkbox__label--error:has(+ .account__error-message:not(:empty)) .account__agreement-inner  {
+  color: var(--color-primary);
+}
+
+.account__agreement:after {
+  content: "";
+  position: fixed;
+  left: 0;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  background: var(--color-primary-67);
+  opacity: 0;
+  visibility: hidden;
+}
+
+.account__agreement:has([id^='user_agreement_opener']:checked):after {
+  opacity: 1;
+  visibility: visible;
+}
+
+.account__agreement-modal {
+  display: block !important;
```

Comment:
Why do we need `!important` here?

Reply:
Because there is `display: none;` inline style in HTML that prevents any layout shifts when the page is loading thereby improving the overall performance